cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -25,11 +25,36 @@ class TLMResponse(TypedDict):
 class TLMOptions(TypedDict):
     """Trustworthy language model options.
 
-    Attributes:
-        max_tokens (int): the maximum number of tokens to generate in the TLM response
+    Parameters
```

Comment:
where are we mentioning that quality-presets determine many of these settings, but specifying them here will over-ride the setting from the quality-preset ?